### PR TITLE
Add Starlink documentation for button

### PR DESCRIPTION
**Pull request by boswelja**
## Proposed change

Adding documentation to the Starlink integration for new switch & button platforms.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/85729
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by bdraco** [comment]
Please limit documentation PRs to match one core PR.

File: source/_integrations/starlink.markdown [hunk 1]
```diff
@@ -4,6 +4,8 @@ description: Instructions on how to integrate Starlink into Home Assistant.
 ha_category:
   - Sensor
   - Binary Sensor
+  - Switch
```
Comment (bdraco) [inline source/_integrations/starlink.markdown]:
```suggestion
```

File: source/_integrations/starlink.markdown [hunk 2]
```diff
@@ -14,6 +16,8 @@ ha_domain: starlink
 ha_platforms:
   - sensor
   - binary_sensor
+  - switch
```
Comment (bdraco) [inline source/_integrations/starlink.markdown]:
```suggestion
```

File: source/_integrations/starlink.markdown [hunk 3]
```diff
@@ -42,3 +46,11 @@ The Starlink integration allows you to integrate your [Starlink](https://www.sta
 - Motors stuck - Whether Dishy is unable to move
 - Thermal throttle - Whether Starlink has reduced performance to avoid overheating
 - Unexpected location - Whether Starlink has detected operation outside of its designated area
+
+### Switch
+
+- Stowed - Controls whether Dishy is stowed
+
```
Comment (bdraco) [inline source/_integrations/starlink.markdown]:
```suggestion
```

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @boswelja 👍 

../Frenck

**Pull request merged**